src/components/StatsPanel/StatsPanel.styles.ts

```diff
@@ -0,0 +1,52 @@
+import { styled } from '@mui/material/styles';
+
+import type { PaletteColor } from '@mui/material/styles';
+
+export const CardContainer = styled('div')(({ theme }) => ({
+  backgroundColor: theme.palette.background.paper,
+  borderRadius: theme.spacing(2.5),
+  padding: theme.spacing(4),
+  display: 'flex',
+  flexDirection: 'column',
+  gap: theme.spacing(3),
+  width: theme.spacing(32.5),
+  boxShadow: theme.shadows[3],
+}));
+
+export const StatItem = styled('div')(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  gap: theme.spacing(1.75),
+}));
+
+export const IconWrapper = styled('div')(({ theme }) => ({
+  width: theme.spacing(6),
+  height: theme.spacing(6),
+  borderRadius: '50%',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  backgroundColor: theme.palette.action.hover,
+  opacity: 0.9,
+}));
+
+export const StatIcon = styled('div')<{ color: keyof import('@mui/material/styles').Palette }>(
```

Comment:
The StatIcon styled component declares the prop type as an inline import: { color: keyof import('@mui/material/styles').Palette }. This inline import in a generic is hard to read and could be simplified and made explicit by importing the Palette type at the top.

**Sugerencia:**
```typescript
Import the Palette type at the top for clarity and use it in the generic. Example:

import type { Palette } from '@mui/material/styles';
export const StatIcon = styled('div')<{ color: keyof Palette }>(...)

This improves readability and maintainability.
```